partcad/src/partcad/user_config.py

```
@@ -10,143 +10,109 @@
             "config.yaml",
         )
         if os.path.exists(config_path):
             try:
-                self.config_obj = yaml.safe_load(open(config_path, "r"))
-                if self.config_obj is None:
-                    self.config_obj = {}
+                self.read_config(open(config_path, "r"))
             except Exception as e:
                 pc_logging.error("ERROR: Failed to parse %s" % config_path)
 
+        if not shutil.which("conda") is None or importlib.util.find_spec("conda") is not None:
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)